src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -276,98 +238,6 @@ internal NodeViewModel NodeViewModel
             get => _node;
         }
         
-        /// <summary>
-        /// Shows or hides the Use Levels and Keep List Structure checkboxes
-        /// in the node chevron popup menu.
-        /// </summary>
-        public bool UseLevelCheckBoxVisibility
-        {
-            get => _port.PortType == PortType.Input;
-        }
-
-        /// <summary>
-        /// Shows or hides the Use Default Value checkbox in the node chevron popup menu.
-        /// </summary>
-        public bool UseDefaultValueCheckBoxVisibility
-        {
-            get => _port.PortType == PortType.Input && DefaultValueEnabled;
-        }
-
-        /// <summary>
-        /// Shows or hides the Break Connections, Hide Wires and UnhideWires buttons in the node chevron popup menu.
-        /// </summary>
-        public bool OutputPortConnectionsButtonsVisible
-        {
-            get => _port.PortType == PortType.Output;
-        }
```

Comment:
Shouldn't these be obsoleted instead of deleting them?

Reply:
They were not in 2.12 so we can just delete them as 2.13 is not released